### DISCUSSION TITLE
Can one script a stick press down?

### DISCUSSION THREAD

**Discussion by ElCapitanP** [nirgoren/Eddienput#45] [Technical support]
It is called R3 and L3 on Playstation, not sure what it would be here. It would be very helpful to script these buttons to record play sequences into the training dummy.

**Comment by nirgoren** [comment]
You can use `'BtnThumbL'` and `'BtnThumbR'` for that in the config file mapping.

**Reply by ElCapitanP** [reply]
Thank you!
